onnxruntime/python/tools/transformers/bert_perf_test.py

```diff
@@ -459,6 +487,18 @@ def parse_arguments():
         help="input name for attention mask",
     )
 
+    parser.add_argument(
+        "--input_tuning_results",
+        default=None,
+        type=str,
+        help="tuning results database to be loaded",
+    )
+    parser.add_argument(
+        "--write_tuning_results",
```

Comment:
--output_tuning_results

Reply:
The original plan is actually as your suggestion. The problem arises when use multiple process, that is `intra_op_num_threads` is not set, there will be multiple process involved. I think we better raise an error in this case, instead of tolerating it?

Reply:
switched to `--output_tuning_results`, if it exists, then insert an infix '.<ctimestamp>.' in the name to workaround.